python/cuvs_bench/pyproject.toml

```diff
@@ -64,7 +61,7 @@ skip = [
 version = { file = "cuvs_bench/VERSION" }
 
 [tool.rapids-build-backend]
-build-backend = "scikit_build_core.build"
+build-backend = "setuptools.build_meta"
```

Comment:
Is this being managed by dependencies.yaml? See the entries on build-system extras for other projects.